helix-zk/src/main/java/org/apache/helix/zkscale/zk/client/SharedZkClient.java

```diff
@@ -0,0 +1,95 @@
+package org.apache.helix.zkscale.zk.client;
+
+
+import java.util.List;
+
+import org.I0Itec.zkclient.IZkConnection;
+//import org.apache.helix.HelixException;
+import org.apache.helix.zkscale.zk.ZkScaleException;
+import org.apache.helix.zkscale.zk.zookeeper.ZkConnection;
+import org.apache.zookeeper.CreateMode;
+import org.apache.zookeeper.data.ACL;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * ZkClient that uses shared ZkConnection.
+ * A SharedZkClient won't manipulate the shared ZkConnection directly.
+ */
+class SharedZkClient extends org.apache.helix.zkscale.zk.ZkClient implements HelixZkClient {
```

Comment:
Should this be public? And could the class path for ZkClient be removed by adjusting import level?